[2023-03-29 13:42:31,548] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T13:42:24.222819+00:00 [queued]>
[2023-03-29 13:42:31,597] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T13:42:24.222819+00:00 [queued]>
[2023-03-29 13:42:31,599] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:42:31,600] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 13:42:31,602] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:42:31,655] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_data_task> on 2023-03-29 13:42:24.222819+00:00
[2023-03-29 13:42:31,666] {standard_task_runner.py:52} INFO - Started process 14804 to run task
[2023-03-29 13:42:31,684] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v2', 'extract_data_task', 'manual__2023-03-29T13:42:24.222819+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpumxupr1s', '--error-file', '/tmp/tmplg2sq4sb']
[2023-03-29 13:42:31,688] {standard_task_runner.py:77} INFO - Job 117: Subtask extract_data_task
[2023-03-29 13:42:31,921] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T13:42:24.222819+00:00 [running]> on host 5ebc180c7cc0
[2023-03-29 13:42:32,142] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v2
AIRFLOW_CTX_TASK_ID=extract_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T13:42:24.222819+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T13:42:24.222819+00:00
[2023-03-29 13:42:35,869] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas_datareader/tiingo.py:234: FutureWarning: In a future version of pandas all arguments of concat except for the argument 'objs' will be keyword-only
  return pd.concat(dfs, self._concat_axis)

[2023-03-29 13:42:35,919] {logging_mixin.py:109} INFO - ingestion from api completed
[2023-03-29 13:42:36,017] {logging_mixin.py:109} INFO - file has been saved at :sp_500_data.csv
[2023-03-29 13:42:36,019] {logging_mixin.py:109} INFO - sp_500_data.csv
[2023-03-29 13:42:36,021] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 13:42:36,061] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v2, task_id=extract_data_task, execution_date=20230329T134224, start_date=20230329T134231, end_date=20230329T134236
[2023-03-29 13:42:36,156] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 13:42:36,276] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
